components/server/src/bitbucket/bitbucket-auth-provider.ts

```diff
@@ -100,14 +100,18 @@ export class BitbucketAuthProvider extends GenericAuthProvider {
 
     protected normalizeScopes(scopes: string[]) {
         const set = new Set(scopes);
-        if (set.has("issue:write")) {
```

Comment:
that was wrong, cf. https://developer.atlassian.com/cloud/bitbucket/bitbucket-cloud-rest-api-scopes/#issue